components/LandingHeader.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div class="landing-header">
+    <EditorLogo size="general" :class="'landing-header__editor-logo'"/>
+    <div class="landing-header__title">
+      Free block-style editor with a universal JSON output
+    </div>
+    <div class="landing-header__description">
+      Editor.js provides maximum power for developers and products staying focused on the end-user experience
+    </div>
+    <Button type="primary" text="Get Started" link="https://editorjs.io/getting-started"/>
+    <UnfairAdvantages :class="'landing-header__ua'"/>
+  </div>
+</template>
+
+<style>
+.landing-header {
+  background: linear-gradient(180deg, #E3F2FF 0%, rgb(250, 253, 255) 98%);
+  padding: 110px 5px 0 5px;
```

Comment:
maybe 10px instead of 5? also, let's move the value to global variable, e.g. `--layout-offset-x`